vault/cluster/cluster.go

```diff
@@ -6,6 +6,8 @@ import (
 	"crypto/x509"
 	"errors"
 	"fmt"
+	"github.com/hashicorp/vault/sdk/helper/certutil"
```

Comment:
would you want to sort the order of the imports? otherwise, LGTM

Reply:
It was a straight revert, i.e. this was how it was before.  I plan to come back to this fix and try again, can address the imports then.